app-ios/Sources/TimetableFeature/TimetableListView.swift

```diff
@@ -66,14 +66,15 @@ public struct TimetableView: View {
                         switch timetableMode {
                         case .list:
                             timetableMode = .grid
-                            switchModeIcon = "list.bullet.indent"
+                            switchModeIcon = .icList
                         case .grid:
                             timetableMode = .list
-                            switchModeIcon = "square.grid.2x2"
+                            switchModeIcon = .icGrid
                         }
                     } label: {
-                        Image(systemName:switchModeIcon).foregroundStyle(AssetColors.Surface.onSurface.swiftUIColor)
-                        .frame(width: 40, height: 40)
+                        Image(switchModeIcon)
+                            .foregroundStyle(AssetColors.Surface.onSurface.swiftUIColor)
+                            .frame(width: 40, height: 40)
```

Comment:
The problem has not occurred because the original image color is onSurface, but I would expect that the color would not be changed unless the template is originally specified in the renderingMode modifier.
```suggestion
                        Image(switchModeIcon)
                            .renderingMode(.template)
                            .foregroundStyle(AssetColors.Surface.onSurface.swiftUIColor)
                            .frame(width: 40, height: 40)
```

I think it would be safer to add this one.
Alternatively, you can also set template-rendering-intent to imageset.

Reply:
@shin-usu 
Thanks your review!
I think these icons should always be template mode, so I set rendering mode on imageset.
[40b52d5](https://github.com/DroidKaigi/conference-app-2024/pull/535/commits/40b52d5a256e5547553642cb9477546ccb65da30)
Please review again 🙏